lib/mas/tasks/firms.rake

```diff
@@ -0,0 +1,14 @@
+namespace :firms do
+  desc 'Index all existing firms'
+  task :index => :environment do
+    puts 'Do you want to index all existing firms? [type `yes` to confirm]'
+    confirmation = STDIN.gets.chomp
```

Comment:
Out of interest why bother with asking for confirmation? Surly by running this you intend to index?